GitUpKit/Views/GIQuickViewController.m

```diff
@@ -235,3 +253,146 @@ - (BOOL)diffFilesViewController:(GIDiffFilesViewController*)controller handleKey
 }
 
 @end
+
+#import "GICommitListViewController.h"
+@interface GIQuickViewControllerWithCommitsList () <GICommitListViewControllerDelegate>
+@property (strong, nonatomic, readonly) GICommitListViewController *leftController;
+@property (strong, nonatomic, readonly) GIQuickViewController *rightController;
+
+@property (strong, nonatomic, readwrite) NSLayoutConstraint *hiddenConstraint;
+@property (strong, nonatomic, readwrite) NSLayoutConstraint *revealedConstraint;
+@end
+
+@implementation GIQuickViewControllerWithCommitsList
+@dynamic commit;
+@dynamic delegate;
+@dynamic list;
```

Comment:
These aren't dynamic...you are implementing the setters but not the getters and they will crash upon access. Implement the getters if it's going to be a property. Otherwise implement them as methods with setters only. What would happen if you called the getter?